### ISSUE TITLE
Chrome and OBS integration automatically cancels a raid

### ISSUE THREAD

**Issue by AgataAah** [night/betterttv#6088]
**Describe the bug**
Chrome and OBS BTTV integration automatically cancels a raid. No matter if I use the /raid command on my Dashboard, regular Chat, OBS Chat or Dashboard Button. As long as BTTV Chrome integration is active in Chrome itself, I can't raid. It doesn't help to switch it of only in OBS. Of course all other integrations, add-ons, chatbots were eliminated to reduce the issue down to BTTV.

**To Reproduce**
Type /raid unsername
BTTV browser integration must be active

**Expected behavior**
Raid will be automatically cancelled by creator a few seconds later

**Screenshots**
![image](https://user-images.githubusercontent.com/130995762/232493512-5fc76b61-d3b4-475c-ac86-851793df3e5f.png)

**Desktop (please complete the following information):**

- OS: Win10
- Browser: Chrome
- Version: 112.0.5615.121

**Comment by night** [comment]
The default value for `Auto Join Raids` is enabled. Did you somehow disable it? Otherwise I'm not sure how you could have changed the default value.

**Comment by AgataAah** [comment]
It indeed does not matter if it's enabled or disabled. The issue happens with both settings, first thing I tried. I didn't have issues with that setting in the past at all.

**Comment by AgataAah** [comment]
> The default value for `Auto Join Raids` is enabled. Did you somehow disable it? Otherwise I'm not sure how you could have changed the default value.

It indeed does not matter if it's enabled or disabled. The issue happens with both settings, first thing I tried. I didn't have issues with that setting in the past at all.

**Comment by night** [comment]
If the setting is enabled, then it won't try to click a button in this area. We have no other code which would click this button.

**Comment by AgataAah** [comment]
> If the setting is enabled, then it won't try to click a button in this area. We have no other code which would click this button.

Thanks for your answer. I tried to break it down to the error with DevTools and the console. I tried every possible combination in the BTTV settings once again. So unchecking "7TV Emotes" for me solved the issue weirdly. That integration gave a lot of errors. After unchecking it, the raids don't automatically get canceled anymore.

![image](https://user-images.githubusercontent.com/130995762/233358454-f3fb3958-71a8-47d5-9f27-be76da33e88c.png)
